frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/utils.ts

```diff
@@ -1,11 +1,12 @@
 import { history } from '@console/internal/components/utils';
 import { VMGenericLikeEntityKind } from 'packages/kubevirt-plugin/src/types/vmLike';
+import { VIRTUALMACHINES_BASE_URL } from '../../../constants/url-params';
 import { getName, getNamespace } from '../../../selectors';
 
 export const redirectToList = (vmi: VMGenericLikeEntityKind, tab?: 'templates' | '' | null) => {
```

Comment:
it looks strange that we need to do this after conforming with console urls and page order.
do we still need breadcrumbs (navigation is flat), do other resource pages use them ?
maybe the console have a "standard" way of creating breadcrumbs we can now use, because we use standrd urls?

Reply:
let's do in a followup